telegraf/telegraf.inputs.dose

```diff
@@ -0,0 +1,41 @@
+##############################################################################
+#######################  DOSE/zcache Metrics  ################################
+##############################################################################
+[[inputs.execd]]
+  command = ["/etc/telegraf/zcache-stats.sh"]
+  name_override = "zcache_stats"
+  signal = "none"
+  data_format = "csv"
+  csv_skip_columns = 1
+  csv_column_names = ["cache_lookup_count","cache_lookup_bytes","idx_access_pendch","idx_access_entry",
+			"idx_access_chunk","idx_access_disk","cache_hits_count","cache_hits_ratio",
+			"cache_insert_count","cache_insert_bytes","insert_source_read","insert_source_write",
+			"insert_source_specr","insert_drops_fullq","insert_drops_lkbusy",
+			"bufbytes_used_demand","bufbytes_used_spec","cache_other_evicts",
+			"cache_other_pending","cache_other_healed","alloc_alloc","alloc_avail",
+			"alloc_free_space","alloc_free_slabs"]
+  csv_column_types = ["int","int","int","int","int","int","int","int","int","int","int","int","int",
+			"int","int","int","int","int","int","int","int","int","int","int"]
```

Comment:
The output of zcache stats has changed recently. This worked well for me:

```
  csv_column_names = ["cache_lookup_count","idx_access_pendch","idx_access_entry","idx_access_chunk",
                        "idx_access_disk","cache_hits_count","cache_hits_bytes","cache_hits_ratio",
                        "cache_insert_count","cache_insert_bytes","insert_source_read","insert_source_write",
                        "insert_source_specr","insert_drops_buffer","insert_drops_alloc","bufbytes_used_demand",
                        "bufbytes_used_spec","cache_other_evicts","cache_other_pending","alloc_alloc",
                        "alloc_avail","alloc_free_space","alloc_free_slabs"]
  csv_column_types = ["int","int","int","int","int","int","int","int","int","int","int","int","int",
                        "int","int","int","int","int","int","int","int","int","int"]
```

Reply:
That's great, thanks for updating this! I have added these changes.